docs/docs/segment-terraform.md

```diff
@@ -0,0 +1,28 @@
+---
+id: terraform
+title: Terraform Context
+sidebar_label: Terraform
+---
+
+## What
+
+Display the currently active Terraform Workspace name.
+
+Note:
+- Will need a terraform binary in your PATH
+- Will only be displayed in directories that contain a `.terraform` subdirectory
+
+## Sample Configuration
+
+```json
+{
+  "type": "terraform",
+  "style": "powerline",
+  "powerline_symbol": "\uE0B0",
+  "foreground": "#000000",
+  "background": "#ebcc34",
+  "properties": {
+    "prefix": " "
```

Comment:
This is the default so that can be omitted :-)

Reply:
Which part? Remove the properties :  {} statement? @JanDeDobbeleer

Reply:
No, prefix defaults to `" "`